src/realm.h

```diff
@@ -1971,6 +1972,35 @@ RLM_API void realm_collection_changes_get_ranges(
     realm_index_range_t* out_modification_ranges_after, size_t max_modification_ranges_after,
     realm_collection_move_t* out_moves, size_t max_moves);
 
+/**
+ * Returns the number of changes occured to the dictionary passed as argument
+ *
+ * @param changes valid ptr to the dictionary changes structure
+ * @param out_deletions_size number of deletions
+ * @param out_insertion_size number of insertions
+ * @param out_modification_size number of modifications
+ */
+RLM_API void realm_dictionary_get_changes(const realm_dictionary_changes_t* changes, size_t* out_deletions_size,
```

Comment:
I don't think this is necessary?

Reply:
see my comment below why this is needed.

Reply:
see my comment below why this is needed.